test/dotnet.Tests/project.json

```diff
@@ -16,6 +16,7 @@
       "target": "project",
       "type": "build"
     },
+    "Microsoft.Extensions.PlatformAbstractions": "1.0.0-rc2-20100",
```

Comment:
This is out of date with what we use in the product:

https://github.com/dotnet/cli/blob/rel/1.0.0/src/Microsoft.DotNet.InternalAbstractions/project.json#L13

```
  "dependencies": {
    "Microsoft.Extensions.PlatformAbstractions": "1.0.0-rc2-20581"
  },
```